DyalogBuild.dyalog

```diff
@@ -1687,6 +1689,7 @@
                   :Trap DEBUG↓0 ⍝ yes, all trap have a halt/ after them - this one doesn't and shouldn't.
                       :If ~0∊⍴type←GetParam'type'
                           :If DyaVersion≥19
+                          :orif _isWin∧DyaVersion≥13
```

Comment:
might be good to add a comment to the effect of saving is available on all platforms from v19.0 and Windows from v13.0